packages/freetype.rb

```diff
@@ -125,18 +122,18 @@ def self.postinstall
     conflictscmd = `grep --exclude #{CREW_META_PATH}#{name}.filelist -Fxf #{CREW_META_PATH}#{name}.filelist #{CREW_META_PATH}*.filelist`
     conflicts << conflictscmd.gsub(/(\.filelist|#{CREW_META_PATH})/, '').split("\n")
     conflicts.reject!(&:empty?)
-    unless conflicts.empty?
-      if conflicts_ok?
-        puts 'Handling conflict with the same file in another package.'.orange
-      else
-        puts 'Error: There is a conflict with the same file in another package.'.lightred
-        @_errors = 1
-      end
-      conflicts.each do |conflict|
-        conflict.each do |thisconflict|
-          singleconflict = thisconflict.split(':', -1)
-          system "sed -i '\\?^#{singleconflict[1]}?d'  #{CREW_META_PATH}/#{singleconflict[0]}.filelist" if @override_allowed.include?(singleconflict[0])
-        end
+    return if conflicts.empty?
+
+    if conflicts_ok?
+      puts 'Handling conflict with the same file in another package.'.orange
+    else
+      puts 'Error: There is a conflict with the same file in another package.'.lightred
+      @_errors = 1
+    end
+    conflicts.each do |conflict|
+      conflict.each do |thisconflict|
```

Comment:
Here is a more elegant way to do it :)
```ruby
# remove nested arrays
conflicts.flatten.each do |conflict|
```

Reply:
I'm a little afraid of touching this code for fear of breaking this.

Reply:
Would it be easier to delete freetype.rb since harfbuzz provides it?